src/content/docs/en/guides/content.mdx

```diff
@@ -4,7 +4,7 @@ description: >-
   Astro is a perfect choice for your content-focused site: blogs, marketing
   sites, portfolios, and more! Author your content directly in your project, or
   connect your CMS of choice.
-i18nReady: false
+i18nReady: true
```

Comment:
`content.mdx` content doesn't seem to be included in the documentation site?

Reply:
@liruifengv i think you are right, but I'll wait for @Yan-Thomas to confirm it before changing it